src/modules/scene/add_question_scene.py

```diff
@@ -113,14 +129,14 @@ def __clear_and_init(self):
         left_x = self.get_screen().get_width() // 4
         right_x = self.get_screen().get_width() // 4 * 3
         self.__option_prompts: list[PromptInput] = [
-            PromptInput(self.get_screen(), "Option A: ",
-                        dimension=(512, 64), top_y=250, top_x=left_x),
-            PromptInput(self.get_screen(), "Option B: ",
-                        dimension=(512, 64), top_y=250, top_x=right_x),
-            PromptInput(self.get_screen(), "Option C: ",
-                        dimension=(512, 64), top_y=435, top_x=left_x),
-            PromptInput(self.get_screen(), "Option D: ",
-                        dimension=(512, 64), top_y=435, top_x=right_x),
+            PromptInput(self.get_screen(), "Option A: ", dimension=(
+                512, 64), top_y=250, top_x=left_x, add_check_box=True),
+            PromptInput(self.get_screen(), "Option B: ", dimension=(
+                512, 64), top_y=250, top_x=right_x, add_check_box=True),
+            PromptInput(self.get_screen(), "Option C: ", dimension=(
+                512, 64), top_y=435, top_x=left_x, add_check_box=True),
+            PromptInput(self.get_screen(), "Option D: ", dimension=(
+                512, 64), top_y=435, top_x=right_x, add_check_box=True),
```

Comment:
I think it would be good to update the styling of the checkbox here!

Reply:
yes good point! we can talk about it today